packages/perf-benchmarks/src/__benchmarks__/engine-dom/benchmark-table-component/tablecmp-update-10th-1k.benchmark.js

```diff
@@ -6,10 +6,10 @@
  */
 
 import { createElement } from 'lwc';
-import Table from 'benchmark/tableComponent';
 
-import Store from '../../tableStore';
-import { insertTableComponent, destroyTableComponent } from '../../utils';
+import Table from 'benchmark/tableComponent';
+import Store from 'benchmark/store';
+import { insertTableComponent, destroyTableComponent } from 'benchmark/utils';
```

Comment:
Looks like we need to either make the following change or rename `benchmark/utils/utils.js` to `benchmark/utils/index.js` and change the other module names to `benchmark/utils`?

```suggestion
import { insertTableComponent, destroyTableComponent } from 'benchmark/utils/utils';
```